apps/user_ldap/lib/User/User.php

```diff
@@ -231,6 +245,60 @@ public function processAttributes($ldapEntry) {
 		}
 		unset($attr);
 
+		//User profile visibility
+		$profileScope = $this->connection->ldapProfileScope;
+		if (is_null($profileScope) || '' === $profileScope || 'unset' === $profileScope) {
```

Comment:
```suggestion
		if ($profileScope === '' || $profileScope === 'unset') {
```
But when is it 'unset' ?

Reply:
When choosen in the advanced configuration tab. The administrator chooses profileScope from unset, v2-private, v2-local, v2-federated, v2-published. If using 'unset' it means the user can set these self in the profile.